src/resources/learningResource/learningResource.model.js

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+
+const learningResourceSchema = new mongoose.Schema({
+  category: {
+    type: String,
+    enum: ['Coding', 'Maths', 'Electronics'],
+    required: true
+  },
+  minAge: {
+    type: Number,
+    required: true
+  },
+  maxAge: {
+    type: Number,
+    required: true
+  },
+  title: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  rating: {
+    type: Number,
+    min: 0,
+    max: 5,
+    required: true
+  },
+  body: String,
+  tags: {
+    type: [String],
+    required: true
+  }
```

Comment:
I'd say tags should be optional since we have the required category already, what do you think?

Reply:
This was an oversight, actually. The required flag only prevents me from setting the field to null - if I omit the field on the request, an empty array is added instead.

I've removed it for now, as I agree that they should be optional anyway.

Reply:
Oh that's interesting... so if the type is an array and required, and empty value passed in will still pass validation, and an empty array put in?

Reply:
Just noticed, I think your most recent commit overwrote the deletion of the `required: true` property

Reply:
Gonna approve anyway, go ahead and change if needed or Squash & Merge if you're happy with it 👍

Reply:
Thanks for spotting that! Not sure how that crept in again...